Sources/Services/AWSSTS/STSClient.swift

```diff
@@ -462,3 +462,12 @@ extension STSClient: STSClientProtocol {
     }
 
 }
+
+extension STSClient {
+    public func presignRequestForGetCallerIdentity(input: GetCallerIdentityInput, expiration: Foundation.TimeInterval) async throws -> URLRequest {
+        guard let presignedRequest = try await input.presign(config: config, expiration: expiration) else {
+            throw ClientError.unknownError("Returned request from input.presign() was nil for the operation GetCallerIdentity.")
```

Comment:
The error message should not make reference to SDK internals; I would say something to the effect of "A presigned URL for \<operation\> could not be created."

Thought of another way, the current message would not make sense anymore if we refactored internally.

Reply:
Changed as suggested.